CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix, prepended
 # rocPRIM project
 project(rocprim LANGUAGES CXX)
 
+# Prefix path
+list(APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/hcc /opt/rocm/hip)
```

Comment:
I am not sure why we need to add this to the PREFIX_PATH when it was working without it.

Reply:
yes, for rocPRIM may not be required. Kept it this change to be in sync. I will remove this change.